images/ose-baremetal-installer.yml

```
@@ -35,5 +35,6 @@ owners:
 - kni-devel-deployment@redhat.com
 konflux:
   vm_override:
-    aarch64: linux-cxlarge/arm64
+    x86_64: linux-d160-m2xlarge/amd64
+    aarch64: linux-d160-m2xlarge/arm64
     s390x: linux-large/s390x
```

```suggestion
    s390x: linux-largecpu/s390x
    ppc64le: linux-largecpu/ppc64le
```
We also have new IBM arches, with higher capacity https://github.com/redhat-appstudio/infra-deployments/blob/main/components/multi-platform-controller/production-downstream/kflux-ocp-p01/host-config.yaml#L14